ddtrace/internal/remoteconfig/client.py

```diff
@@ -366,12 +384,19 @@ def _load_new_configurations(self, applied_configs, client_configs, payload):
 
             try:
                 log.debug("Load new configuration: %s. content ", target)
-                callback(config, config_content)
+                if isinstance(callback, RemoteConfigCallBackAfterMerge):
```

Comment:
Can't this aggregation be performed in ASM? Isn't there enough information in each config to perform it outside of this class? If so, perhaps we can just change the callback mechanism to just do aggregation for each product?

Reply:
No, I think we can't, We can aggregate the data in ASM but we don't know in ASM side when we should report aggregate data or if it's a new payload. 

In other hand, we need to aggregate more than one product. With this approach, each team (ASM, APM, Debugger) decide what mechanism of callback is better for their needs through this new Callback class. I think it's the best approach to adapt RC callbacks for the team needs without modify the logic of other teams payloads